build/meson/programs/meson.build

```diff
@@ -42,7 +42,8 @@ endif
 
 export_dynamic_on_windows = false
 # explicit backtrace enable/disable for Linux & Darwin
-if not use_backtrace
+execinfo = cc.has_header('execinfo.h', required: use_backtrace)
+if execinfo.found()
```

Comment:
This changed from `not use_backtrace` to `execinfo.found()` which seems inverted?

Reply:
Yeah mistake